src/SFA.DAS.ApprenticeCommitments.Web.UnitTests/Features/ConfirmIdentityEnforcedSteps.cs

```diff
@@ -80,9 +79,8 @@ public void ThenRedirectTheUserToTheHomePage()
         [Then("redirect the user to the TermsOfUse page")]
         public void ThenRedirectTheUserToTermsOfUse()
         {
-            _context.Web.Response.Should().Be2XXSuccessful();
-            _context.ActionResult.LastPageResult.Should().NotBeNull();
-            _context.ActionResult.LastPageResult.Model.Should().BeOfType<TermsOfUseModel>();
+            _context.Web.Response.Should().Be302Found();
```

Comment:
I haven't got to the code yet (it's ⬇️) but I would expect that the redirect logic is handled in another library now, and so shouldn't be tested here.

Reply:
It's not, we simply are checking for the user being redirected (changed the code to just check for a 302)